src/InstrumentationEngine.ProfilerProxy/dllmain.cpp

```diff
@@ -63,7 +63,7 @@ namespace ProfilerProxy
     /*
      * This function returns whether the current process is running under Wow64
      */
-    static HRESULT IsWow64(_Out_ BOOL* pIsWow64)
+    static HRESULT IsWow64(_In_ CEventLogger* pEventLogger, _Out_ BOOL* pIsWow64)
```

Comment:
>CEventLogger* [](start = 32, length = 13)

Consider passing these by reference instead of by pointer, so that there is no risk of an AV.